exec/eval_test.go

```diff
@@ -37,6 +38,10 @@ func (testExecutor) Reader(*Task, int) sliceio.ReadCloser {
 	panic("not implemented")
 }
 
+func (testExecutor) Eventer() eventlog.Eventer {
```

Comment:
Optional: `testExecutor` seems to be satisfying `Executor` with `testing.T.Name`, which is a bit weird, and maybe it should define its own.

Reply:
I went a step further and removed the field altogether, as it has been defunct since bc0b8d738a54f20ea453b96647f3f6a1ec735109.